xdp-loader/xdp-loader.c

```diff
@@ -285,6 +285,25 @@ int do_status(const void *cfg, __unused const char *pin_root_path)
 	return iface_print_status(opt->iface.ifindex ? &opt->iface : NULL);
 }
 
+static const struct cleanopt {
+	struct iface iface;
+} defaults_clean = {};
+
+static struct prog_option clean_options[] = {
+	DEFINE_OPTION("dev", OPT_IFNAME, struct cleanopt, iface,
+		      .positional = true, .metavar = "[ifname]",
+		      .help = "Clean up detached program links for [ifname] (default all interfaces)"),
+	END_OPTIONS
+};
+
+int do_clean(const void *cfg, __unused const char *pin_root_path)
+{
+	const struct cleanopt *opt = cfg;
+
+	printf("Cleaning up detached XDP program links...\n");
```

Comment:
This should include whether we're cleaning all interfaces or just the one. Also, ending with an ellipsis implies there will be more output coming, so just end with a period.